client/my-sites/theme/main.jsx

```diff
@@ -523,6 +521,23 @@ const ThemeSheetWithOptions = ( props ) => {
 	);
 };
 
+const themeDetailsFromTheme = ( theme ) => {
```

Comment:
I think by our convention, this name should use a verb -- one option that makes sense here is again `get`: `getThemeDetailsFromTheme`. In the long run, we should move it to `state/theme/utils`, call it `normalizeTheme`, and use it right in the reducer that corresponds to `receiveTheme` to only store themes after normalization.